SignalMessaging/contacts/SystemContactsFetcher.swift

```diff
@@ -36,7 +36,8 @@ class ContactsFrameworkContactStoreAdaptee: ContactStoreAdaptee {
         CNContactPhoneNumbersKey as CNKeyDescriptor,
         CNContactEmailAddressesKey as CNKeyDescriptor,
         CNContactPostalAddressesKey as CNKeyDescriptor,
-        CNContactViewController.descriptorForRequiredKeys()
+        CNContactViewController.descriptorForRequiredKeys(),
+        CNContactVCardSerialization.descriptorForRequiredKeys()
```

Comment:
Because we're now serializing contacts to/from card using the iOS vcard implementation.  That serialization will fail if its input is missing what it considers required fields.  We're not serializing to vcard yet because we haven't implementation sharing contacts from Signal iOS yet.  I briefly implemented it then disabled it as the design evolved and it was failing without this.